### PR TITLE
Added plan facility update persister

### PR DESCRIPTION
**Pull request by Saloni-eGov**

### CODE REVIEW

File: health/egov-persister/plan-service-persister.yml [hunk 1]
```diff
@@ -494,3 +494,29 @@ serviceMaps:
             - jsonPath: $.Plan.auditDetails.lastModifiedBy
 
             - jsonPath: $.Plan.auditDetails.lastModifiedTime
+
+    - version: 1.0.0
+      description: Updates the plan facility
+      fromTopic: update-plan-facility
+      isTransaction: true
+      queryMaps:
+        - query: UPDATE plan_facility_linkage SET residing_boundary = ?, service_boundaries = ?, additional_details = ?, active = ?,  last_modified_by=?, last_modified_time=? WHERE id = ?;
```
Comment (Priyanka-eGov) [inline health/egov-persister/plan-service-persister.yml:503]: last_modified_by=?, space is inconsistent

File: health/egov-persister/plan-service-persister.yml [hunk 2]
```diff
@@ -494,3 +494,29 @@ serviceMaps:
...
+          basePath: $.PlanFacility
+          jsonMaps:
+            - jsonPath: $.PlanFacility.residingBoundary
+
+            - jsonPath: $.PlanFacility.serviceBoundaries
```
Comment (Priyanka-eGov) [inline health/egov-persister/plan-service-persister.yml:508]: why is serviceBoundaries of type JSONB, in the db design its of type TEXT

**Review by Priyanka-eGov — CHANGES_REQUESTED** [review]

File: health/egov-persister/plan-service-persister.yml (hunk 1 above)
Comment (Saloni-eGov) [reply health/egov-persister/plan-service-persister.yml:503]: Corrected the space

File: health/egov-persister/plan-service-persister.yml (hunk 2 above)
Comment (Saloni-eGov) [reply health/egov-persister/plan-service-persister.yml:508]: Corrected to text

File: health/egov-persister/plan-service-persister.yml [hunk 3]
```diff
@@ -494,3 +494,29 @@ serviceMaps:
             - jsonPath: $.Plan.auditDetails.lastModifiedBy
 
             - jsonPath: $.Plan.auditDetails.lastModifiedTime
+
+    - version: 1.0.0
+      description: Updates the plan facility
+      fromTopic: update-plan-facility
+      isTransaction: true
+      queryMaps:
+        - query: UPDATE plan_facility_linkage SET residing_boundary = ?, service_boundaries = ?, additional_details = ?, active = ?,  last_modified_by= ?, last_modified_time= ? WHERE id = ?;
```
Comment (Priyanka-eGov) [inline health/egov-persister/plan-service-persister.yml:503]: inconsistent spacing.
last_modified_by= ?, last_modified_time= ?

**Review by Priyanka-eGov — CHANGES_REQUESTED** [review]

File: health/egov-persister/plan-service-persister.yml [hunk 4]
```diff
@@ -533,3 +533,29 @@ serviceMaps:
             - jsonPath: $.PlanFacility.auditDetails.lastModifiedTime
```
Comment (Priyanka-eGov) [inline health/egov-persister/plan-service-persister.yml:535]: too many new lines. Remove unnecessary new lines.

**Review by Priyanka-eGov — CHANGES_REQUESTED** [review]

File: health/egov-persister/plan-service-persister.yml (hunk 4 above)
Comment (Saloni-eGov) [reply health/egov-persister/plan-service-persister.yml:535]: done

**Pull request merged**